### PR TITLE
refactor(form): refactor various components to glimmer

### PR DESCRIPTION
**Pull request by anehx**
Refactor the following form components:

- cf-form
- cf-content
- cf-field-value
- cf-field
- cf-field/errors
- cf-field/info
- cf-field/input

### CODE REVIEW

**Comment by anehx** [comment]
@czosel @velrest @luytena please add your review if possible

File: addon/components/cf-content.js [hunk 1]
```diff
@@ -39,103 +38,79 @@ import { parseDocument } from "ember-caluma/lib/parsers";
...
-    if (this.document) this.document.destroy();
-    if (this.navigation) this.navigation.destroy();
-  },
+  @queryManager apollo;
 
   /**
    * The uuid of the document to display
    *
    * @argument {String} documentId
    */
```
Comment (czosel) [inline addon/components/cf-content.js:51]: Are the arguments picked up by addon-docs even without the variable declaration?

Comment (anehx) [reply addon/components/cf-content.js:51]: Yes

File: addon/components/cf-field-value.js [hunk 2]
```diff
@@ -40,19 +38,18 @@ export default Component.extend({
...
-        },
-        "node.fileValue"
-      );
-      window.open(downloadUrl, "_blank");
-    },
-  },
-});
+  @action
+  async download(fileAnswerId) {
+    const { downloadUrl } = await this.apollo.query(
```
Comment (czosel) [inline addon/components/cf-field-value.js:45]: Why `query` instead of `watchQuery`?

Comment (czosel) [reply addon/components/cf-field-value.js:45]: Ah I see, for the download we only need to fetch once.

**Review by czosel — APPROVED** [review]
Nice! :+1:

**Pull request merged**